data/json/mapgen/basement/basement_meth.json

```diff
@@ -0,0 +1,232 @@
+        "R": "f_shower",
+        "A": "f_air_filter",
+        "O": "f_standing_tank",
+        "p": "f_safe_l",
+        "u": "f_table",
+        "n": "f_chair",
+        "Y": [ [ "f_crate_c", 2 ], [ "f_crate_o", 3 ], [ "f_null", 10 ] ]
+      },
+      "liquids": { "W": { "liquid": "water_clean", "amount": [ 0, 100 ] } },
+      "place_loot": [
+        { "group": "meth_ingredients", "x": [ 18, 21 ], "y": 20, "repeat": [ 10, 20 ] },
+        { "group": "hazmat_suits", "x": 17, "y": 13 },
+        { "group": "hazmat_masks", "x": 17, "y": 13 },
+        { "group": "exotic_books", "x": 15, "y": 17 },
+        { "group": "methlab", "x": 22, "y": [ 15, 16 ], "chance": 75, "repeat": [ 1, 3 ] },
+        { "group": "methlab", "x": 22, "y": [ 19, 20 ], "chance": 75, "repeat": [ 1, 3 ] },
+        { "group": "guns_shotgun_common", "x": 22, "y": 12 },
+        { "group": "ammo_shotgun_common", "x": 22, "y": 12, "amount": [ 7, 30 ] },
+        { "group": "home_hw", "x": 13, "y": 20, "chance": 40, "repeat": [ 1, 3 ] },
+        { "group": "home_hw", "x": [ 21, 22 ], "y": 5, "chance": 40, "repeat": [ 1, 3 ] },
+        { "group": "religious_books", "x": 15, "y": 17 },
+        { "group": "magazines", "x": 15, "y": [ 12, 13 ], "repeat": [ 3, 5 ] },
+        { "group": "bed", "x": [ 8, 9 ], "y": [ 6, 7 ], "chance": 90, "repeat": [ 1, 3 ] },
+        { "group": "dresser", "x": 8, "y": 5, "chance": 85, "repeat": [ 2, 5 ] },
+        { "group": "dresser", "x": 8, "y": 8, "chance": 85, "repeat": [ 2, 5 ] },
+        { "group": "dresser", "x": 13, "y": [ 5, 6 ], "chance": 85, "repeat": [ 2, 5 ] },
+        { "group": "laundry", "x": 19, "y": 5, "chance": 50, "repeat": [ 1, 10 ] },
+        { "group": "laundry", "x": 19, "y": 6, "chance": 50, "repeat": [ 1, 10 ] },
+        { "group": "softdrugs", "x": [ 9, 10 ], "y": 12, "chance": 75, "repeat": [ 1, 3 ] },
+        { "group": "cleaning", "x": 11, "y": 10, "chance": 90, "repeat": [ 1, 3 ] },
+        { "group": "cleaning", "x": 16, "y": 5, "chance": 90, "repeat": [ 1, 3 ] }
+      ],
+      "place_item": [
+        { "x": 18, "y": 19, "item": "meth" },
+        { "x": 21, "y": 12, "item": "meth", "amount": [ 5, 20 ] },
+        { "x": 15, "y": [ 12, 13 ], "item": "meth", "chance": 90 },
+        { "x": 21, "y": 12, "item": "money_bundle", "amount": [ 3, 10 ] },
+        { "x": 15, "y": 5, "item": "corpse" },
+        { "x": 18, "y": 19, "item": "chemistry_set" },
+        { "x": 11, "y": 10, "item": "towel" },
+        { "x": 16, "y": 5, "item": "bleach" },
+        { "x": 16, "y": 5, "item": "knife_steak" }
+      ],
+      "place_monster": [
+        { "monster": "mon_zombie_hazmat", "x": [ 19, 21 ], "y": [ 15, 20 ], "chance": 50 },
+        { "monster": "mon_zombie_hazmat", "x": [ 13, 14 ], "y": [ 11, 16 ], "chance": 50 },
+        { "monster": "mon_zombie_rot", "x": 15, "y": 6, "chance": 30 },
+        { "monster": "mon_zombie_common", "x": [ 13, 14 ], "y": [ 11, 17 ], "chance": 90, "repeat": [ 1, 3 ] }
```

Comment:
There is no monster named `mon_zombie_common`. Did you mean `mon_zombie`?

Reply:
Yeah, I must have pushed old changes.